src/ui/zabapgit_js_common.w3mi.data.js

```diff
@@ -2512,7 +2517,7 @@ function restoreScrollPosition(){
   window.sessionStorage.setItem("scrollTop", 0);
 }
 
-function memoizeScrollPosition(fn){
+function memorizeScrollPosition(fn){
```

Comment:
As non native speaker it's hard to distinguish when to use `memoize` vs. `memorize` therefore I made it a habit to always use `memoize` while coding as there seems to be some subtile differences.
https://wikidiff.com/memoize/memorize
It's also built in other languages e.p. as python decorator. 

So it wasn't a typo in first place, but surely can be adjusted.